### PR TITLE
Hotfix fldyn python tests clean up

### PR DESCRIPTION
**Pull request by rubenzorrilla**
Now the tests take around 100 seconds in FullDebug. @jcotela as we discussed, maybe it is a good idea to move the 3D ones to the validation suite. 

In any case, the deprecated tests have been removed. Besides, the format is now unique for all the tests.

### CODE REVIEW

**Comment by RiccardoRossi** [comment]
hi @rubenzorrilla thx for your effort, however 100s is still far too long.

it should take let's say 10s in full debug (small test suite)

**Comment by rubenzorrilla** [comment]
The allSuite is the one that takes 100 s.

**Review by jcotela — CHANGES_REQUESTED** [review]
Hi @rubenzorrilla I agree with the changes you have made but I'd like to ask you one more thing. Could you re-organize the existing tests into the small, nitghtly, all and validation sutites, and report how long each of them runs (in fulldebug?).

Also, a small request from my side: could you correct the long-standing typo in `AssambleTestSuites` (-> Assemble)

File: applications/FluidDynamicsApplication/tests/test_FluidDynamicsApplication.py [hunk 1]
```diff
@@ -60,8 +50,12 @@ def AssambleTestSuites():
     smallSuite.addTest(DarcyChannelTest('testDarcyDensity'))
     smallSuite.addTest(EmbeddedReservoirTest('testEmbeddedReservoir2D'))
     smallSuite.addTest(EmbeddedReservoirTest('testEmbeddedSlipReservoir2D'))
-    smallSuite.addTest(EmbeddedAusasCouetteTest('testEmbeddedAusasCouette2D'))
-    smallSuite.addTest(EmbeddedAusasCouetteTest('testEmbeddedDevelopmentCouette2D'))
+    smallSuite.addTest(EmbeddedCouetteTest('testEmbeddedCouette2D'))
```
Comment (philbucher) [inline applications/FluidDynamicsApplication/tests/test_FluidDynamicsApplication.py]: I recommend using `smallSuite.addTests(KratosUnittest.TestLoader().loadTestsFromTestCases([EmbeddedReservoirTest]))`

this detects all the tests in `EmbeddedReservoirTest` automatically and they don't have to be added manually in this file

It happened several times in StructuralMechanics that developers wrote their tests but forgot to add them

Comment (rubenzorrilla) [reply applications/FluidDynamicsApplication/tests/test_FluidDynamicsApplication.py]: @philbucher thx for the advise! However, I guess that this functionality adds all the tests in a file to a unique suite (please correct me if I'm wrong). The point is that in our case some of them go to one suite and some of them to another.

Comment (philbucher) [reply applications/FluidDynamicsApplication/tests/test_FluidDynamicsApplication.py]: I think you are right, in this case you have to add it manually afaik

**Review by jcotela — APPROVED** [review]

**Comment by jcotela** [comment]
@philbucher hopefully this will help with #1915

**Pull request merged**